README.md

```diff
@@ -17,7 +17,7 @@ Node server hosted on AWS, mainly used for REST endpoints.
 
 You can get these credentials from Avaer.
 
-3. Create a `cert/` folder. Create 2 new files inside: `fullchain.pem` and `privkey.pem`. Ask Avaer for the certificates.
+3. Run `npm run script:generateCerts`
```

Comment:
I suspect the browser will not like this; the certs need to not be self-signed.

Reply:
The browser is fine with it, I run Chrome with --ignore-certificate-errors during development.
Also seems not to be the root of the access denied issue. So for now it *might* be OK to use locally signed certs (and much more secure) but this warrants further investigation.

Reply:
> I run Chrome with --ignore-certificate-errors during development.
Don't do that.